src/Nethermind/Nethermind.TxPool/Comparison/ByHashTxComparer.cs

```diff
@@ -23,7 +23,7 @@ namespace Nethermind.TxPool.Comparison
     /// <summary>
     /// Compares <see cref="Transaction"/>s based on <see cref="Transaction.Hash"/> identity. No two different signed transactions will be same.
     /// </summary>
-    public class ByHashTxComparer : IComparer<Transaction>
+    public class ByHashTxComparer : IComparer<Transaction>, IEqualityComparer<Transaction>
```

Comment:
For example here we have Comparer not Compare.

Reply:
Yes was thinking about it, guys used different convention here, lets leave it for now and see which will read better after some time